### ISSUE TITLE
[FEATURE] Add support for ignoring children of arbitrary tags

### ISSUE THREAD

**Issue by reteps** [yeonjuan/html-eslint#381]
I have html like this:

````
<markdown>
# Hello, world!

```cpp{4-6,9}
#include <iostream>

class Example {
    Example() {
        std::cout << "Hello, world!" << std::endl;
    }

    Example(std::string name) {
        std::cout << "Hello, " << name << std::endl;
    }
};
```
</markdown>
````

I would like to be able to mark the `markdown` node as special, so that I don't get false positives, i.e. on the `<iostream>` tag.

**Comment by yeonjuan** [comment]
Hello @reteps , thank you for the suggestion. It seems like a useful feature for certain cases.

**Comment by yeonjuan** [comment]
Work is in progress. To implement this feature, I'm going to add a `customTags` option to the parser. If the `customTags` option is set, it will parse the matching tag's children as raw text (i.e., it won't treat `<` as the start of a tag).

https://github.com/yeonjuan/es-html-parser/pull/32/files#diff-1cc56e8d7a270ffdc1ae861ddadae386d1f382cbe0717a7e5821831179a35007R113-R129

**Comment by yeonjuan** [comment]
Hi @reteps A version with the `rawContentTags` option added has been released. `v0.44.0`.
- https://html-eslint.org/docs/integrating-template-engine#rawcontenttags
